reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md

```diff
@@ -179,59 +178,88 @@ Set-NetFirewallRule -Name "WINRM-HTTP-In-TCP-PUBLIC" -RemoteAddress Any
 The `Set-NetFirewallRule` cmdlet is exported by the NetSecurity module.
 
 > [!NOTE]
-> In Windows PowerShell 2.0, on computers running server versions
-> of Windows, `Enable-PSRemoting` creates firewall rules that allow remote
-> access on private, domain and public networks. On computers running
-> client versions of Windows, `Enable-PSRemoting` creates firewall rules
-> that allow remote access only on private and domain networks.
+> In Windows PowerShell 2.0, on computers running server versions of Windows,
+> `Enable-PSRemoting` creates firewall rules that allow remote access on
+> private, domain and public networks. On computers running client versions of
+> Windows, `Enable-PSRemoting` creates firewall rules that allow remote access
+> only on private and domain networks.
 
-### HOW TO ENABLE A FIREWALL EXCEPTION BY USING A GROUP POLICY
+### How to enable a firewall exception by using a group policy
 
 ```
 ERROR:  ACCESS IS DENIED
 
 or
 
 ERROR: The connection to the remote host was refused. Verify that the
-WS-Management service is running on the remote host and configured to
-listen for requests on the correct port and HTTP URL.
+WS-Management service is running on the remote host and configured to listen
+for requests on the correct port and HTTP URL.
 ```
 
 To enable a firewall exception for in all computers in a domain, enable the
-"Windows Firewall: Allow local port exceptions" policy in the following
-Group Policy path:
+**Windows Firewall: Allow local port exceptions** policy in the following Group
+Policy path:
 
-Computer Configuration\\Administrative Templates\\Network
-\\Network Connections\\Windows Firewall\\Domain Profile
+```
+Computer Configuration\Administrative Templates\Network
```

Comment:
Same here... this is a single path... should it be one line or the second line indented to indicate such?

Reply:
stet